client/src/main/java/connection/chain/ChainConfigFactory.java

```diff
@@ -2,10 +2,9 @@
 
 public class ChainConfigFactory {
 
-  private ChainConfigFactory() {
-  }
+private ChainConfigFactory() {}
 
-  public static Chain configureChainOfResponsibilities() {
+public static Chain configureChainOfResponsibilities() {
```

Comment:
javadoc

Reply:
Not my code, will be added by Bartek in his pull request